src/backup.ts

```diff
@@ -0,0 +1,520 @@
+    databaseAdmin.spanner.admin.database.v1.ICreateBackupMetadata;
+}
+
+export type CreateBackupResponse = [
+  Backup,
+  CreateBackupGaxOperation,
+  databaseAdmin.longrunning.IOperation
+];
+
+/**
+ * IBackup structure with backup state enum translated to string form.
+ */
+type IBackupTranslatedEnum = TranslateEnumKeys<
+  databaseAdmin.spanner.admin.database.v1.IBackup,
+  'state',
+  typeof databaseAdmin.spanner.admin.database.v1.Backup.State
+>;
+
+export type GetMetadataResponse = [IBackupTranslatedEnum];
+type GetMetadataCallback = RequestCallback<IBackupTranslatedEnum>;
+
+type UpdateExpireTimeCallback = RequestCallback<
+  databaseAdmin.spanner.admin.database.v1.IBackup
+>;
+
+type DeleteCallback = RequestCallback<void>;
+
+/**
+ * The {@link Backup} class represents a Cloud Spanner backup.
+ *
+ * Create a `Backup` object to interact with or create a Cloud Spanner backup.
+ *
+ * @class
+ *
+ * @example
+ * const {Spanner} = require('@google-cloud/spanner');
+ * const spanner = new Spanner();
+ * const instance = spanner.instance('my-instance');
+ * const backup = instance.backup('my-backup');
+ */
+class Backup {
+  id: string;
+  formattedName_: string;
+  instanceFormattedName_: string;
+  request: <T, R = void>(
+    config: RequestConfig,
+    callback: RequestCallback<T, R>
+  ) => void;
+  constructor(instance: Instance, name: string) {
+    this.request = instance.request;
+    this.instanceFormattedName_ = instance.formattedName_;
+    this.formattedName_ = Backup.formatName_(instance.formattedName_, name);
+    this.id = this.formattedName_.split('/').pop() || '';
+  }
+
+  create(
+    databasePath: string,
+    expireTime: PreciseDate
+  ): Promise<CreateBackupResponse>;
+  create(
+    databasePath: string,
+    expireTime: PreciseDate,
+    options?: CallOptions
+  ): Promise<CreateBackupResponse>;
+  create(
+    databasePath: string,
+    expireTime: PreciseDate,
+    callback: CreateBackupCallback
+  ): void;
+  create(
+    databasePath: string,
+    expireTime: PreciseDate,
+    options: CallOptions,
+    callback: CreateBackupCallback
+  ): void;
+  /**
+   * @typedef {array} CreateBackupResponse
+   * @property {Backup} 0 The new {@link Backup}.
+   * @property {Operation} 1 An {@link Operation} object that can be used to check
+   *     the status of the request.
+   * @property {object} 2 The full API response.
+   */
+  /**
+   * @callback CreateBackupCallback
+   * @param {?Error} err Request error, if any.
+   * @param {Backup} backup The new {@link Backup}.
+   * @param {Operation} operation An {@link Operation} object that can be used to
+   *     check the status of the request.
+   * @param {object} apiResponse The full API response.
+   */
+  /**
+   * Create a backup.
+   *
+   * @method Backup#create
+   * @param {string} databasePath The path of the database.
+   * @param {PreciseDate} expireTime The expiry time of the backup.
+   * @param {object} [options] Configuration object.
```

Comment:
Please document as "gaxOptions" and include link to gax found in my previous comment that showed an example.